source/sync/configure/sync-settings.txt

```diff
@@ -61,15 +61,48 @@ update the backend schema by changing the client model.
 To learn more about how the Realm Object Schemas map to the App Services Schemas
 when using Development Mode, refer to :ref:`Data Model Mapping <sync-data-model-mapping>`.
 
-For more information about modifying synced object schemas, including how to
-handle breaking changes, see: :ref:`<synced-schema-overview>`.
+For more information about modifying synced object schemas, see:
+:ref:`<synced-schema-overview>`.
 
 .. important:: Disable Development Mode for Production Apps
    
    Development mode is a development utility that is not suitable for 
    production use. Make sure that you turn off Development Mode before you 
    make your app accessible in a production environment.
 
+.. _dev-mode-breaking-changes:
+
+Breaking Changes
+````````````````
+
+*Applies to App Services Apps created after August 9th, 2023.* 
+
+App Services Apps in Development Mode that were created after August 9th, 2023
+can make :ref:`breaking changes <breaking-change-quick-reference>` to synced
+object schemas from client code.
+
+.. note:: Apps created before August 9th, 2023
+   
+   For Apps created before August 9th, 2023, you need to update your
+   object schema in the App Services UI. For details, see
+   :ref:`<synced-schema-overview>`
+
+To make a breaking change from client code:
+
+#. Change your local object model and let it sync to your App.
+#. Delete your local realm and data.
+#. Open a new realm with your updated object model.
```

Comment:
"and let it sync to your app" is a little misleading imo. Whenever they make a change and try to run their mobile app again, it will fail to open with an error message.

The flow is to change their schema, then delete the realm before opening it with the new schema changes.

Reply:
Ooh, great point. Thank you! I removed the sync bit in my latest (had to revert because the feature didn't go live).

Reply:
Latest PR is here: https://github.com/mongodb/docs-app-services/pull/548